### PR TITLE
Docs improvements for channel

### PR DESCRIPTION
**Pull request by douglaz**
See https://github.com/lightningdevkit/rust-lightning/pull/2100#discussion_r1136016923
It's blocked on https://github.com/lightningdevkit/rust-lightning/pull/2077

### CODE REVIEW

**Comment by dunxen** [comment]
I believe this will have little impact and won't change things too much based on what I'm currently fixing up with #2077. So we don't need to consider it blocked.

**Comment by dunxen** [comment]
At least not that PR specifically.

File: lightning/src/ln/channel.rs [hunk 1]
```diff
@@ -4565,9 +4565,11 @@ impl<Signer: WriteableEcdsaChannelSigner> Channel<Signer> {
...
 	}
 
-	/// Guaranteed to be Some after both ChannelReady messages have been exchanged (and, thus,
-	/// is_usable() returns true).
+	/// Guaranteed to be [`Some`] after both [`ChannelState::ChannelReady`] messages have been exchanged
+	/// (and, thus, [`is_usable()`] returns true).
 	/// Allowed in any state (including after shutdown)
+	///
+	/// [`is_usable()`]: Self::is_usable
 	pub fn get_short_channel_id(&self) -> Option<u64> {
```
Comment (dunxen) [inline lightning/src/ln/channel.rs:4573]: Ah wait. My bad. This will end up changing. Will let you know once I've unblocked you.

**Comment by douglaz** [comment]
> This should be un-draft-able now. Any desire to pick it back up @douglaz?

Done. No clue if the comments are still valid though.

**Comment by douglaz** [comment]
Done rebasing

**Review by valentinewallace — APPROVED** [review]

**Pull request merged**